packages/core/src/layout/breakpoint/breakpoint.service.ts

```diff
@@ -1,21 +1,25 @@
-import { BreakpointObserver, Breakpoints } from '@angular/cdk/layout';
 import { Injectable, signal } from '@angular/core';
 
 @Injectable({
   providedIn: 'root'
 })
 export class BreakpointService {
   isHandset = signal(false);
```

Comment:
On pourrait rendre la propriété `isHandset` readonly pour s'assurer qu'elle ne soit pas muter dans le code.
```suggestion
  private _isHandset = signal(false);

  get isHandset() {
    return this._isHandset.asReadonly();
  }
```